connection.go

```diff
@@ -1305,12 +1305,19 @@ func (conn *Connection) ConfiguredTimeout() time.Duration {
 // OverrideSchema sets Schema for the connection.
 func (conn *Connection) OverrideSchema(s *Schema) {
```

Comment:
```suggestion
func (conn *Connection) SetSchema(s Schema) {
```

We also need to ensure that we use a copy of data to avoid changes in the internal implementation from the external code:

```Go
conn.OverrideSchema(schema)
schema.SomePublicField = append(schema.SomePublicField, foo)
```

We need to ensure that it does not affect a result of usage of a Connection object to make it same.

Reply:
Updated.